internal/app/main.go

```diff
@@ -9,7 +9,7 @@ const (
 	appVersion         = "v3.3.0"
 	tempFilesDir       = ".helmsman-tmp"
 	defaultContextName = "default"
-	resourcePool       = 10
+	resourcePool       = 100
```

Comment:
But why? :-)

Reply:
there' something off about the concurrency that runs multiple helm commands, i don't think its speeding things up much, but i'll revert this for now